OptimizelySDK.Tests/OptimizelyTest.cs

```diff
@@ -3412,5 +3413,16 @@ public void TestAfterDisposeAPIsShouldNotCrash()
         }
 
         #endregion
+
+        #region Test GetOptimizelyConfig
+        [Test]
+        public void TestGetOptimizelyConfig()
+        {
+            ProjectConfig actualConfig = OptimizelyMock.Object.ProjectConfigManager?.GetConfig();
+            OptimizelyConfig optimizelyConfig = OptimizelyMock.Object.GetOptimizelyConfig();
+            Assert.AreEqual(optimizelyConfig.Revision, actualConfig.Revision);
```

Comment:
Shouldn't we assert everything ? Revision doesn't suffice this test.

Reply:
Unit tests are in progress. Noman also mentioned it in the PR description.

Reply:
@zashraf1985 OK, u asked me to review on slack that's why i was expecting unit test.

Reply:
I should wait for unit tests completed for review